docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -101,15 +110,14 @@ spec:
 EOF
 ```
 
-```console
-$ kubectl get issuers  -n send-receive-project -o wide send-receive-root-issuer --watch
-NAME                       READY   STATUS   AGE
-send-receive-root-issuer   True             45s
+```{"stage":"cert-manager"}
+$ kubectl wait issuer --all --for=condition=Ready --namespace=send-receive-project
+issuer.cert-manager.io/send-receive-root-issuer condition met
```

Comment:
even doing this automated testing process has caused us to improve the tutorial to use more dynamic commands and expose the conditions. great!

Reply:
I think this dynamic command can be improved, it's waiting for every issuer to be up, and we could make it so it only waits for the one we've just created.